articles/native-platforms/phonegap/01-login.md

```diff
@@ -4,24 +4,23 @@ description: This tutorial will show you how to use the Auth0 Phonegap SDK to ad
 ---
 
 ## Phonegap Tutorial
-
-<%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-cordova',
-  pkgBranch: 'master',
-  pkgPath: 'examples/phonegap-basic-sample',
-  pkgFilePath: 'examples/phonegap-basic-sample/www/js',
-  pkgType: 'js'
-}) %>
-
-**Otherwise, if you already have an existing application, please follow the steps below.**
-
 ::: panel-info System Requirements
 This tutorial and seed project have been tested with the following:
 
 * NodeJS 4.3
 * Phonegap 5.5
 :::
 
+<%= include('../../_includes/_package', {
```

Comment:
@mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.